backend/constants.js

```diff
@@ -0,0 +1,11 @@
+import {assoc} from 'ramda';
+
+import {getSetting} from './settings';
+
+export const COOKIE_OPTIONS = {
+    secure: getSetting('SSL_ENABLED'),
+    path: getSetting('WEB_BASE_PATHNAME'),
+}
+
+export const ACCESS_TOKEN_COOKIE_OPTIONS = assoc(
+   'maxAge', getSetting('ACCESS_TOKEN_AGE'), COOKIE_OPTIONS);
```

Comment:
Right now these will be evaluated on import which might not actually work in the test environment (where we change the settings on runtime). So, we might want to change to something like:
```
export function getCookieOptions() {
    return {
        secure: getSetting('SSL_ENABLED'),
        path: getSetting('WEB_BASE_PATHNAME')
    }
}
```